django/contrib/admin/actions.py

```diff
@@ -33,8 +33,8 @@ def delete_selected(modeladmin, request, queryset):
 
     # Populate deletable_objects, a data structure of all related objects that
     # will also be deleted.
-    deletable_objects, model_count, perms_needed, protected = get_deleted_objects(
-        queryset, opts, request.user, modeladmin.admin_site, using)
+    deletable_objects, model_count, perms_needed, protected = modeladmin.get_deleted_objects(
+        queryset, opts, request, using)
```

Comment:
To help conform to our style guidelines, put the ) on the next line and adding a comma after using.